src/portable/options.rs

```diff
@@ -431,6 +434,31 @@ pub struct Info {
     pub version: Option<ver::Filter>,
 }
 
+#[derive(Debug, Clone, Copy)]
+pub enum ShowCredentialsType {
+    JSON,
+    InsecureDSN,
+}
+
+#[derive(EdbClap, Clone, Debug)]
+pub struct ShowCredentials {
+    #[clap(long, possible_values=&[
+    "json", "insecure-dsn",
+    ])]
+    pub _type: Option<ShowCredentialsType>
```

Comment:
We have `--json` everywhere. So I would either use `--json/--insecure-dsn` or use `--format=json/--format=insecure-dsn` and add a `--json` shortcut for the former.

Also we don't use `--json` by default anywhere (Do we?). Maybe we want some human-readable text/table with hidden password by default?

It's also interesting why we have `insecure-dns` but plain `json`, while `json` also has the same password.